scroll-verifier/deploy_l1/00_scroll_verifier.ts

```diff
@@ -0,0 +1,38 @@
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import 'dotenv/config';
+
+const GATEWAY_URLS = {
+  scrollDevnetL1: 'http://localhost:8089/{sender}/{data}.json',
+  goerli: 'https://scroll-gateway-worker.ens-cf.workers.dev/{sender}/{data}.json',
+  // Point to localhost if you want to test locally
+  // sepolia: 'http://localhost:8080/{sender}/{data}.json'
+  sepolia: 'https://scroll-sepolia-gateway-worker.ens-cf.workers.dev/{sender}/{data}.json',
+};
+
+const ROLLUP_ADDRESSES = {
+  sepolia: '0x64cb3A0Dcf43Ae0EE35C1C15edDF5F46D48Fa570',
```

Comment:
This should be the final version used in sepolia: https://sepolia.etherscan.io/address/0xE0BfA7f3B06A9589A914BE09Ba0E5671f481A722#code.